rpcs3/rpcs3qt/tooltips.h

```diff
@@ -1,4 +1,4 @@
-﻿#pragma once
+#pragma once
 
```

Comment:
Iirc

Reply:
Yes because we didn't have the /utf-8 command for msvc so it didn't interpret stuff as utf-8 unless it had the bom marker.
Now that the option has been added bom or no bom shouldn't matter but it's better to remove it, mainly because of the gcc bug(where it ignores #pragma once on headers with BOM).